src/test/kotlin/dev/stalla/builder/validating/ValidatingPodcastindexLocationBuilderTest.kt

```diff
@@ -0,0 +1,89 @@
+package dev.stalla.builder.validating
+
+import assertk.all
+import assertk.assertAll
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isFalse
+import assertk.assertions.isNotNull
+import assertk.assertions.isNull
+import assertk.assertions.isTrue
+import assertk.assertions.prop
+import dev.stalla.builder.PodcastindexLocationBuilder
+import dev.stalla.model.aPodcastindexGeographicLocation
+import dev.stalla.model.aPodcastindexOpenStreetMapElement
+import dev.stalla.model.anEpisodePodcastindexLocation
+import dev.stalla.model.podcastindex.PodcastindexLocation
+import org.junit.jupiter.api.Test
+
+internal class ValidatingPodcastindexLocationBuilderTest {
+
+    @Test
+    internal fun `should not build an Podcastindex Location with when all the fields are missing`() {
+        val locationBuilder = ValidatingPodcastindexLocationBuilder()
+
+        assertAll {
+            assertThat(locationBuilder).prop(PodcastindexLocationBuilder::hasEnoughDataToBuild).isFalse()
+
+            assertThat(locationBuilder.build()).isNull()
+        }
+    }
+
+    @Test
+    internal fun `should not build an Podcastindex Location with when the name field is missing`() {
+        val locationBuilder = ValidatingPodcastindexLocationBuilder()
+            .geo(aPodcastindexGeographicLocation())
+
+        assertAll {
+            assertThat(locationBuilder).prop(PodcastindexLocationBuilder::hasEnoughDataToBuild).isFalse()
+
+            assertThat(locationBuilder.build()).isNull()
+        }
+    }
+
+    @Test
+    internal fun `should build an Podcastindex Location with all the mandatory fields`() {
+        val locationBuilder = ValidatingPodcastindexLocationBuilder()
+            .name("name")
+
+        assertAll {
+            assertThat(locationBuilder).prop(PodcastindexLocationBuilder::hasEnoughDataToBuild).isTrue()
+
+            assertThat(locationBuilder.build()).isNotNull().all {
+                prop(PodcastindexLocation::name).isEqualTo("name")
+                prop(PodcastindexLocation::geo).isNull()
+                prop(PodcastindexLocation::osm).isNull()
+            }
+        }
+    }
+
+    @Test
+    internal fun `should build an Podcastindex Location with all the added entries to its fields`() {
```

Comment:
```suggestion
    internal fun `should build a Podcastindex Location with all fields filled in correctly`() {
```

Or something along those lines. Doesn't seem we're actually testing this for the  previous models?

Reply:
Hm, in most builder tests we do ("...when there are all fields"). Just not for the OpenStreetMap implementation in this PR.